spec/core/ics-026-routing-module/README.md

```diff
@@ -447,6 +447,12 @@ function handleChanOpenInit(datagram: ChanOpenInit) {
     datagram.portIdentifier,
     datagram.counterpartyPortIdentifier
   )
+  // SYNCHRONOUS: the following calls happen synchronously with the call above
+  // ASYNCHRONOUS: the module callback will be called at a time later than the channel handler
+  // in this case, the channel identifier will be stored with a sentinel value so it is not taken
```

Comment:
With this you mean that a sentinel value is stored in the path `channelEnds/ports/{datagram.portIdentifier}/channels/{channelIdentifier}`, right? Maybe it's more explicit to mention it like this?